ui/webui/src/components/Error.jsx

```diff
@@ -52,16 +58,23 @@ const addExceptionDataToReportURL = (url, exception) => {
     );
     newUrl.searchParams.append(
         "comment",
-        "Instructions about what to fill here and which logs tarball to attach"
+        "Installer WebUI Critical Error:\n" + exception.name + ": " + exception.message + "\n\n" + _("Please attach the file /tmp/webui.log to the issue.")
     );
     return newUrl.href;
 };
 
 export const CriticalError = ({ exception, isBootIso, reportLinkURL }) => {
     const reportURL = addExceptionDataToReportURL(reportLinkURL, exception);
+    const [logContent, setLogContent] = useState("");
+
+    useEffect(() => {
+        const syslogFile = cockpit.file("/tmp/syslog");
+        syslogFile.read().then(content => setLogContent(content));
+    }, []);
 
     const openBZIssue = (reportURL) => {
-        window.open(reportURL, "_blank");
+        cockpit.file("/tmp/webui.log").replace(logContent)
```

Comment:
On first look this looks like something that the backend would generate, but I guess that if we want to include data from the front end, this might be a better way. 

Or we might end up with a hybrid  eventually where a backend & frontend generated files get merged.

Reply:
Maybe I should name it journal.log because in fact it just contains the content of journal. If we add some more information we can name it accordingly. Or is it better to have some specific name (like webui.log) from the beginning... not sure.